wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/wicket-ajaxdownload.js

```diff
@@ -108,29 +108,32 @@
 						var disposition = xhr.getResponseHeader("Content-Disposition");
 						if (disposition) {
 							var matches = /filename[^;=\n]*=(([""]).*?\2|[^;\n]*)/.exec(disposition);
-							if (matches != null && matches[1]) {
+							if (matches !== null && matches[1]) {
```

Comment:
Let's add this .js file to the ones checked by JSLint in `testing/js-tests/Gruntfile.js

Reply:
Wasn't aware of this file
Just have checked `../../wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/wicket-ajaxdownload.js` is already added

Reply:
Strange that it didn't break the build due to `!=`.